instrumentation/okhttp/okhttp-3.0/library/src/main/java/io/opentelemetry/instrumentation/okhttp/v3_0/OkHttpAttributesExtractor.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.okhttp.v3_0;
+
+import io.opentelemetry.instrumentation.api.instrumenter.http.HttpAttributesExtractor;
+import io.opentelemetry.semconv.trace.attributes.SemanticAttributes;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+final class OkHttpAttributesExtractor extends HttpAttributesExtractor<Request, Response> {
+  @Override
+  protected @Nullable String method(Request request) {
+    return request.method();
+  }
+
+  @Override
+  protected @Nullable String url(Request request) {
+    return request.url().toString();
+  }
+
+  @Override
+  protected @Nullable String target(Request request) {
+    return null;
+  }
+
+  @Override
+  protected @Nullable String host(Request request) {
+    return request.url().host();
+  }
+
+  @Override
+  protected @Nullable String route(Request request) {
+    return null;
+  }
+
+  @Override
+  protected @Nullable String scheme(Request request) {
+    return request.url().scheme();
+  }
+
+  @Override
+  protected @Nullable String userAgent(Request request) {
+    return request.header("User-Agent");
```

Comment:
```suggestion
    return request.header("user-agent");
```

Probably a good pattern to use lowercase to make sure it's not case sensitive